test/cli/disabled-read-ahead/args

```diff
@@ -2,4 +2,5 @@
 --table-input-path $IN_DIR/table.csv
 --table-output-path $OUT_DIR/table.csv
 --table-output-manifest-path $OUT_DIR/table.csv.manifest
---ahead-blocks 0
\ No newline at end of file
+--input-size-threshold "0B"
```

Comment:
All inputs in the tests are under the threshold, so threshold is disabled in old tests.